bitcoin/build.rs

```diff
@@ -1,4 +1,4 @@
-const MSRV_MINOR: u64 = 56;
+const MSRV_MINOR: u64 = 63;
```

Comment:
I think we should be able to pull this from `Cargo.toml`. Maybe there's even an env var set by cargo.

Reply:
In my local CI I extract them from clippy.toml, which is more reliably set than the `version` field in the main Cargo.toml.

Hopefully there is a way to extract this without bringing the full power and dep tree of toml-rs into our build dependency tree.

Reply:
`clippy` should pull it from Cargo.toml too. We should check and migrate.

> Hopefully there is a way to extract this without bringing the full power and dep tree of toml-rs into our build dependency tree.

grep-like approach would be enough IMO. We're looking for a specific field in a specific table.